internal/staticanalysis/parsing/js/babel-parser.js

```diff
@@ -42,21 +56,27 @@ function logIdentifierOrPrivateName(identifierType, node) {
     }
 
     let name = identifierNode.name;
-    let pos = locationString(identifierNode)
+    let pos = locationString(identifierNode);
 
     if (identifierNode.name === undefined) {
         console.log("Error: undefined identifier name at pos " + pos);
     }
 
-    logJSON("Identifier", identifierType, name, pos, null, null);
+    logJSON("Identifier", identifierType, name, pos);
 }
 
 function logLiteral(literalType, value, pos, inArray, extra = null) {
     if (value === undefined) {
         console.log("Error: undefined literal value at pos " + pos);
         return;
     }
-    logJSON("Literal", literalType, value, pos, inArray, extra);
+
+    if (extra === null) {
+        extra = {"array": inArray};
```

Comment:
nit: it's probably a bit cleaner to do:

```javascript

if (extra === null) {
  extra = {};
}

extra.array = inArray;
```

To avoid some duplication and make the logic a bit easier to understand.

Reply:
Nice, I'll add this in another PR